CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via [issue on our bug tracker](https://trac.osgeo.org/osgeolive/report/1),
+[OSGeoLive mailing list](https://lists.osgeo.org/mailman/listinfo/osgeolive), or any other method with the owners of this repository before making a change. 
+
+By submitting you are agreeing to make your content available via our [open license policy](https://live.osgeo.org/en/copyright.html).
+
+In the same manner, when translating on Transifex, you are agreeing to make your translations following those licence terms.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+
+## Pull Request Process
+
+1. TO-DO
+
+## Translation process
+### Join a translation team and translate
+
+1. Create an account or login on [Transifex](https://www.transifex.com/osgeo/osgeolive/)
+2. Request joining a translation team
+3. Translate ! 
+4. Please consider adding your self to our [contributor list](https://github.com/OSGeo/OSGeoLive-doc/blob/master/contributors.csv), if you don't know how to use git, please contact a maintainer.
+
```

Comment:
Right now we have the egg-hen problem.
egg = sign contirbution agreement
hen = start translating, we will use your translations

the hen is already given, but we didnt have the egg